styles.css

```diff
@@ -0,0 +1,126 @@
+html {
+  background-color: rgba(229, 229, 229, 0.2);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "poppins", sans-serif;
+  width: 375px;
+  height: 718px;
+  border-bottom-left-radius: 79px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.home-container {
+  padding: 0 24px;
+  background-image: url("./icons/background.jpg");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position-y: 25px;
+}
+
+.logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 58px 24px 0 24px;
+  background-color: rgba(255, 255, 255, 1);
+  margin: 0;
+}
+
+.logo h1 a {
+  text-decoration: none;
+  color: rgba(96, 112, 255, 1);
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+}
+
+.contact-icons {
+  display: flex;
+  align-items: center;
+  align-content: center;
+  justify-content: space-between;
+  width: 170.17px;
+  list-style-type: none;
+  padding-bottom: 0;
+  padding-left: 0;
+}
+
+.contact-icons img {
+  width: 20px;
+  height: 20px;
+}
+
+.logo a img {
+  width: 18px;
+  height: 12px;
+}
+
+#peace img {
+  width: 14.17px;
+}
+
+#github img {
+  width: 17.99px;
+}
+
+#m img {
+  height: 16px;
+}
+
+.main-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 128px 0 12px 0;
+  margin: 0;
+}
```

Comment:
- [ ] .main-info has a lot of padding on top, on the design 128px padding on top is until the logo itself, in your case it is 128px + space under the the logo: 
<img width="346" alt="Screenshot 2022-05-31 at 14 24 54" src="https://user-images.githubusercontent.com/77038610/171152752-601d93af-c56f-4c2f-ad72-7cf6feb300c1.png">

I recommend setting padding top to `100px`, assuming that the space under the logo is around `20px` and space below h2 tag is around `7px`

After you are done setting the proper padding for navbar and this section, your page should look like this: 

Your design  | After
------------- | -------------
<img width="357" alt="Screenshot 2022-05-31 at 14 29 05" src="https://user-images.githubusercontent.com/77038610/171153538-f47a79a4-d0bb-4e21-b9ab-219bd833567b.png"> | <img width="349" alt="Screenshot 2022-05-31 at 14 27 47" src="https://user-images.githubusercontent.com/77038610/171153307-2a454050-65bc-4e33-930f-a8f620e2cf89.png">